src/conversions.jl

```diff
@@ -844,7 +844,7 @@ function convert(::Type{PyAny}, o::PyObject)
     try
         T = pytype_query(o)
         if T == PyObject && is_pyjlwrap(o)
-            return unsafe_pyjlwrap_to_objref(o.o)
+            return unsafe_pyjlwrap_to_objref(PyPtr(o))
```

Comment:
`unsafe_pyjlwrap_to_objref` should probably just have a method that takes a `PyObject`

Reply:
Done.